recipes-config/systemd-resource-control/systemd-resource-control.bb

```diff
@@ -0,0 +1,86 @@
+
+SRC_URI += " \
+    file://LICENSE \
+    file://slice.audio.conf \
+    file://slice.connectivity.conf \
+    file://slice.coreservices.conf \
+    file://slice.dlt.conf \
+    file://slice.media.conf \
+    file://slice.phone.conf \
+    file://slice.speech.conf \
+    file://slice.swdl.conf \
+    file://slice.ui-apps.conf \
+    file://slice.ui.conf \
+    file://slice.vehiclefunctions.conf \
+    file://slice.video.conf \
+    file://service.connectivity-manager.conf \
+"
+
+S = "${WORKDIR}"
+
+inherit allarch
+
+do_install_append() {
+    install -d ${D}${systemd_unitdir}/system/
+
+    # Slices
+    install -d ${D}${systemd_unitdir}/system/audio.slice.d/
+    install -m 0644 ${WORKDIR}/slice.audio.conf ${D}${systemd_unitdir}/system/audio.slice
+
+    install -d ${D}${systemd_unitdir}/system/connectivity.slice.d/
+    install -m 0644 ${WORKDIR}/slice.connectivity.conf ${D}${systemd_unitdir}/system/connectivity.slice
+
+    install -d ${D}${systemd_unitdir}/system/coreservices.slice.d/
+    install -m 0644 ${WORKDIR}/slice.coreservices.conf ${D}${systemd_unitdir}/system/coreservices.slice
+
+    install -d ${D}${systemd_unitdir}/system/dlt.slice.d/
+    install -m 0644 ${WORKDIR}/slice.dlt.conf ${D}${systemd_unitdir}/system/dlt.slice
+
+    install -d ${D}${systemd_unitdir}/system/media.slice.d/
+    install -m 0644 ${WORKDIR}/slice.media.conf ${D}${systemd_unitdir}/system/media.slice
+
+    install -d ${D}${systemd_unitdir}/system/phone.slice.d/
+    install -m 0644 ${WORKDIR}/slice.phone.conf ${D}${systemd_unitdir}/system/phone.slice
+
+    install -d ${D}${systemd_unitdir}/system/speech.slice.d/
+    install -m 0644 ${WORKDIR}/slice.speech.conf ${D}${systemd_unitdir}/system/speech.slice
+
+    install -d ${D}${systemd_unitdir}/system/swdl.slice.d/
+    install -m 0644 ${WORKDIR}/slice.swdl.conf ${D}${systemd_unitdir}/system/swdl.slice
+
+    install -d ${D}${systemd_unitdir}/system/ui-apps.slice.d/
+    install -m 0644 ${WORKDIR}/slice.ui-apps.conf ${D}${systemd_unitdir}/system/ui-apps.slice
+
+    install -d ${D}${systemd_unitdir}/system/ui.slice.d/
+    install -m 0644 ${WORKDIR}/slice.ui.conf ${D}${systemd_unitdir}/system/ui.slice
+
+    install -d ${D}${systemd_unitdir}/system/vehiclefunctions.slice.d/
+    install -m 0644 ${WORKDIR}/slice.vehiclefunctions.conf ${D}${systemd_unitdir}/system/vehiclefunctions.slice
+
+    install -d ${D}${systemd_unitdir}/system/video.slice.d/
+    install -m 0644 ${WORKDIR}/video.slice ${D}${systemd_unitdir}/system/video.conf
+
+    # Service appends
+    install -m 0644 ${WORKDIR}/service.connectivity-manager.conf ${D}${systemd_unitdir}/system/connectivity.slice.d/
+}
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/services:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/slices:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/systemd-resource-control/:"
+FILES_${PN} += " \
+    ${systemd_unitdir}/system/ \
+    ${systemd_unitdir}/system/audio.slice.d/ \
```

Comment:
Add the missing connectivity-manager.service.d folder.

Reply:
but it's not a slice

Reply:
You are correct, but the folder is created by this recipe, i.e. ${D}${systemd_unitdir}/system/connectivity-manager.service.d/service.conf. Shouldn't it be part of FILES_${PN} then?

Reply:
Furthermore, the slice override folders are not installed because we don't install anything in them, so I assume they shall be removed from this variable?